### PR TITLE
handle not covered unit cases

### PR DESCRIPTION
**Pull request by EricBlack**

### CODE REVIEW

File: AElf.Sdk.CSharp.Tests/EventTests.cs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+{
+    public class EventTests
+    {
+        // Lot of types data should not be defined including: byte, sbyte, char, short, ushort, decimal, float and double.
+        [Fact]
+        public void Parse_EventTest()
+        {
+            var eventDemo = new EventDemo()
+            {
+               //init index value
```
Comment (loning) [inline AElf.Sdk.CSharp.Tests/EventTests.cs]: make them in different events and then check them

Comment (EricBlack) [reply AElf.Sdk.CSharp.Tests/EventTests.cs]: have updated code and test with different events.

File: AElf.Sdk.CSharp.Tests/EventCollection.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        [Indexed] public long IndexLong {get;set;}
+
+        public Address NonIndexAddress {get;set;}
+        public string NonIndexString {get;set;}
+        public int NonIndexInt {get;set;}
+    }
+
+    public class NotSupportEvent : Event
+    {
+        [Indexed] public double IndexDouble {get;set;}
```
Comment (loning) [inline AElf.Sdk.CSharp.Tests/EventCollection.cs]: should define in different classes, each only contains one 
 public class NotSupportDoubleIndexEvent : Event
    {
        [Indexed] public double IndexDouble {get;set;}
}
 public class NotSupportDoubleEvent : Event
    {
        public double IndexDouble {get;set;}
}

Comment (EricBlack) [reply AElf.Sdk.CSharp.Tests/EventCollection.cs]: updated not support event type case one by one.

**Pull request merged**